tests/main.cpp

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file main.cpp
+ * @author Jeffin Sam
+ */
+
+#include <iostream>
+#include <mlboard/core.hpp>
+
+//#define CATCH_CONFIG_MAIN  // catch.hpp will define main().
+#define CATCH_CONFIG_RUNNER  // We will define main().
+#include "catch.hpp"
+
+int main(int argc, char** argv)
+{
+  // GOOGLE_PROTOBUF_VERIFY_VERSION;
+  std::cout << "Start testing " << std::endl;
+  // google::protobuf::ShutdownProtobufLibrary();
```

Comment:
Can this be removed, if not maybe we should add a comment that explains why this is here.

Reply:
I think you mistakenly resolved this.

Reply:
https://github.com/mlpack/mlboard/pull/6/commits/f8f66cee586c55ae000ea97e92d9cefe8e315728